lib/fetchPR.ts

```diff
@@ -1,34 +1,39 @@
 import getRocksetClient from "./rockset";
 import { PRData } from "./types";
 
-export default async function fetchPR(pr: string): Promise<PRData[]> {
+export default async function fetchPR(pr: string): Promise<PRData> {
   const rocksetClient = getRocksetClient();
-  const PRQuery = await rocksetClient.queryLambdas.executeQueryLambda(
-    "commons",
-    "prs_query",
-    "eea6932288753f6d",
-    {
-      parameters: [
-        {
-          name: "pr",
-          type: "int",
-          value: pr,
-        },
-      ],
-    }
-  );
-  const shaSet = new Set();
-  const results: PRData[] = [];
+  const [prQuery, commitHistoryQuery] = await Promise.all([
+    rocksetClient.queryLambdas.executeQueryLambdaByTag(
+      "commons",
+      "pr_query",
+      "latest",
```

Comment:
Is this what we want to do going forward? You can change the queries without review and break the hud right?

Reply:
mmm I'm fine to change it back. That was mostly for development.